### PR TITLE
OSS release with multi-arch image build

### PR DESCRIPTION
**Pull request by leiyiz**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:
Support for multi-arch build

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -18,15 +18,11 @@ steps:
...
-    - GIT_TAG=${_GIT_TAG}
     - PULL_BASE_REF=${_PULL_BASE_REF}
     - PROJECT=${_STAGING_PROJECT}
     args:
-    - build-image-and-push
+    - build-and-push-multi-arch
 substitutions:
-  # _GIT_TAG will be filled with a git-based tag for the image, of the form vYYYYMMDD-hash, and
-  # can be used as a substitution.
-  _GIT_TAG: '12345'
```
Comment (mattcary) [inline cloudbuild.yaml:29]: does git_tag need to be plumbed through to STAGINGVERSION? I don't know if cloud build uses that when it releases (?)

Comment (leiyiz) [reply cloudbuild.yaml:29]: GIT_TAG seems to be no longer in use anymore, `STAGINGVERSION` reads from `PULL_BASE_REF`

**Comment by mattcary** [comment]
killer.

/lgtm

File: Makefile [hunk 2]
```diff
@@ -61,55 +60,72 @@ webhook:
...
+		    --build-arg WEBHOOKBINARY=$(WEBHOOKBINARY) \
+			--build-arg STAGINGVERSION=$(STAGINGVERSION) \
+			--build-arg BUILDPLATFORM=linux/amd64 \
+			--build-arg TARGETPLATFORM=linux/amd64 \
+			-f ./cmd/webhook/Dockerfile \
+			-t $(WEBHOOK_STAGINGIMAGE):$(STAGINGVERSION) --push .; \
 		}
 
 push-webhook-image: webhook-image
-	{                                       \
```
Comment (saikat-royc) [inline Makefile:80]: let's not remove the push-webhook-image rules?

**Pull request merged**